File: src/main/java/sample/controller/dialogs/About.java
package sample.controller.dialogs;

import javafx.scene.control.Dialog;
import javafx.scene.web.WebEngine;
import javafx.scene.web.WebView;
import javafx.stage.Window;

public class About extends Dialog {
    public About() {
        super();

        super.setTitle("About");
        super.setResizable(true);

        WebView view = new WebView();
        WebEngine engine = view.getEngine();

        engine.load("https://jedlimlx.github.io/Cellular-Automaton-Viewer/");
        super.getDialogPane().setContent(view);

        // Allows closing with close button
        Window window = super.getDialogPane().getScene().getWindow();
        window.setOnCloseRequest(event -> window.hide());
    }
}

File: src/main/kotlin/application/controller/dialogs/About.kt
package application.controller.dialogs

import javafx.scene.web.WebView
import javafx.scene.web.WebEngine
import javafx.scene.layout.GridPane
import application.model.Giffer
import javafx.scene.control.SpinnerValueFactory.IntegerSpinnerValueFactory
import javafx.scene.layout.VBox
import javafx.scene.chart.NumberAxis
import javafx.scene.chart.LineChart
import javafx.scene.chart.XYChart.Series
import javafx.scene.chart.XYChart
import javafx.stage.FileChooser
import java.io.FileWriter
import java.io.IOException
import java.util.logging.LogManager
import javafx.beans.value.ObservableValue
import javafx.beans.property.SimpleStringProperty
import application.model.SymmetryGenerator
import javafx.event.EventHandler
import javafx.scene.control.*
import javafx.stage.WindowEvent

class About : Dialog<Any?>() {
    init {
        super.setTitle("About")
        super.setResizable(true)

        val view = WebView()
        val engine = view.engine
        engine.load("https://jedlimlx.github.io/Cellular-Automaton-Viewer/")

        super.getDialogPane().content = view

        // Allows closing with close button
        val window = super.getDialogPane().scene.window
        window.onCloseRequest = EventHandler { window.hide() }
    }
}